test/unit/gateways/rapyd_test.rb

```diff
@@ -76,7 +76,7 @@ def test_successful_purchase_without_cvv
     response = stub_comms(@gateway, :ssl_request) do
       @gateway.purchase(@amount, @credit_card, @options)
     end.check_request do |_method, _endpoint, data, _headers|
-      assert_match(/"number":"4242424242424242","expiration_month":"9","expiration_year":"2024","name":"Longbob Longsen/, data)
+      assert_match(/"number":"4242424242424242","expiration_month":"9","expiration_year":"2025","name":"Longbob Longsen/, data)
```

Comment:
Instead of manually modifying the year you could use something like this:

```ruby
      assert_match(/"number":"4242424242424242","expiration_month":"9","expiration_year":"#{Time.now.year+1}","name":"Longbob Longsen/, data)

```

Reply:
Oh yes, you are right, let me update it. thanks